Assets/Scripts/Environment/LabLightsController.cs

```diff
@@ -1,14 +1,50 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.Rendering;
 
 public class LabLightsController : MonoBehaviour
 {
+    public Texture2D[] darkLightmapDir, darkLightmapColor;
+    public Texture2D[] brightLightmapDir, brightLightmapColor;
+    public Material lightOn;
+    public Material lightOff;
+
+    private LightmapData[] darkLightMap, brightLightmap;
```

Comment:
```suggestion
    private LightmapData[] _darkLightMap, _brightLightmap;
```
just to keep things consistent